tests/conftest.py

```diff
@@ -8,10 +8,19 @@
 import sys
 import time
 from contextlib import ExitStack, contextmanager
-from typing import TYPE_CHECKING, Dict, Iterable, List
+from typing import TYPE_CHECKING, Callable, Dict, Iterable, Iterator, List, Optional
 from unittest.mock import patch
 
+import py.path
 import pytest
+
+# Config will be availble from the public API next release:
```

Comment:
Better to say explicitly what that next release will be (i.e. will be available in >XXX). It’s difficult to know what the next release this line is referring to in the future.

Reply:
Sounds good. I changed this to "# Config will be available from the public API in pytest >= 7.0.0:" in the latest revision.